tests/unittests/tests-flashpage/Makefile.include

```diff
@@ -0,0 +1 @@
+FEATURES_REQUIRED += periph_flashpage
```

Comment:
Is the plan to move this to another application in the future? Because this makes the unittests not runnable on all platforms (which they should).

Reply:
Hmm yes, actually ~20 are note run because of this.

Reply:
Blacklisting or `FEATURES_OPTIONAL` works as well.

Reply:
Blacklisting would have generate the same issue. I guess for `FEATURES_OPTIONAL` this test would need to be excluded somehow?

Reply:
(We don't need to do the moving in this PR, it could be done as a follow-up)

Reply:
(I just want to know if this is planned or not)

Reply:
why excluded? If its optional then only `BOARD`s that have it will pull the code, and only `native` seems to need to do so for this test.

Reply:
But does the test work or compile if the feature is not provided?

Reply:
This would work as well:

```diff
diff --git a/tests/unittests/tests-flashpage/Makefile.include b/tests/unittests/tests-flashpage/Makefile.include
index 95277f63b4..e69de29bb2 100644
--- a/tests/unittests/tests-flashpage/Makefile.include
+++ b/tests/unittests/tests-flashpage/Makefile.include
@@ -1 +0,0 @@
-FEATURES_REQUIRED += periph_flashpage
diff --git a/tests/unittests/tests-flashpage/tests-flashpage.c b/tests/unittests/tests-flashpage/tests-flashpage.c
index 00ddd6c7cc..0ff627d768 100644
--- a/tests/unittests/tests-flashpage/tests-flashpage.c
+++ b/tests/unittests/tests-flashpage/tests-flashpage.c
@@ -32,6 +32,10 @@
 
 #include "periph/flashpage.h"
 
+#if defined(BOARD_NATIVE)
+char _native_flash[FLASHPAGE_SIZE * FLASHPAGE_NUMOF];
+#endif
+
 static void test_flashbase_addr(void)
 {
     void *addr
```

Reply:
Edited with `#if defined(BOARD_NATIVE) && !defined(MODULE_PERIPH_FLASHPAGE)`, in which case it someone was to include the module it would still work.

Reply:
Is this preferred?

Reply:
I think that is cleaner and does not involve so much black magic ;-) to be honest. However, use `IS_USED` for the `MODULE_` macro ;-).

Reply:
Regardless, with that I still would prefer to have it in its own test, because it requires board-dependent resources.

Reply:
So let's go with this solution for now, as it does not require yet another Murdock run.